src/templates/common/medallia/index.test.tsx

```diff
@@ -20,8 +15,78 @@ jest.mock('@/lib/utils/medallia', () => ({
   setWindowVaSurvey: jest.fn(),
 }))
 
+// Mock recordEvent
+jest.mock('@/lib/analytics/recordEvent', () => ({
+  recordEvent: jest.fn(),
+}))
+
+interface MedalliaEvent extends CustomEvent {
+  detail: {
+    Form_Type?: string
+    Form_ID?: string
+    Feedback_UUID?: string
+    Content?: string
+  }
+}
```

Comment:
🤔 Would this benefit from pulling out into another module, or do you suppose this is the only place we'd need to define the shape of a Medallia event? (Something something useful for defining mock data for testing, but the implementation details are hidden from actual usage..?)

Reply:
Good question, I would bet some decent money that we won't do this anywhere else ever again. So I'm 50/50 on it.